src/components/titlePanel/createTitlePanel.ts

```diff
@@ -0,0 +1,90 @@
+import { VNodeProperties } from '@dojo/interfaces/vdom';
```

Comment:
@smhigley will have good suggestions for what's needed. You can omit focus / tabindex stuff for now until we get a focus manager.